src/Nowruz/pages/setting/sections/account/account.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button } from 'src/components/atoms/button/button';
+import { Input } from 'src/Nowruz/modules/general/components/input/input';
+import { Modal } from 'src/components/templates/modal/modal';
+import { FeaturedIcon } from 'src/Nowruz/modules/general/components/featuredIcon-new';
+
+import { useRef, useState,FormEvent } from 'react';
+import css from './account.module.scss';
+import {
+    User,Organization,identities
+  } from 'src/core/api';
+import { SearchDropdown } from 'src/Nowruz/modules/general/components/SearchDropdown';
+
+import { useSelector } from 'react-redux';
+import store, { RootState } from 'src/store';
+import { Textarea } from 'src/components/atoms/textarea/textarea';
+import { deleteAccount } from 'src/pages/delete-profile/delete-profile.service';
```

Comment:
Here the pages folder is related to V2 and it is going to be removed. Do not import from it. Moreover we don't use '.service' files anymore.